packages/web/src/scss/components/UNSTABLE_Footer/README.md

```diff
@@ -0,0 +1,204 @@
+          <li>
+            <a href="www.example.com">Link</a>
+          </li>
+          <li>
+            <a href="www.example.com">Link</a>
+          </li>
+          <li>
+            <a href="www.example.com">Link</a>
+          </li>
+        </ul>
+      </nav>
+
+      <nav aria-labelledby="footer-part-two" class="mb-600 mb-tablet-800">
+        <h3 class="typography-body-large-text-bold mb-600" id="footer-part-two">Section headline</h3>
+
+        <ul class="Stack Stack--hasSpacing" style="--stack-spacing: var(--spirit-space-500)">
+          <li>
+            <a href="www.example.com">Link</a>
+          </li>
+          <li>
+            <a href="www.example.com">Link</a>
+          </li>
+          <li>
+            <a href="www.example.com">Link</a>
+          </li>
+          <li>
+            <a href="www.example.com">Link</a>
+          </li>
+        </ul>
+      </nav>
+
+      <nav aria-labelledby="footer-part-three" class="mb-600 mb-tablet-800">
+        <h3 class="typography-body-large-text-bold mb-600" id="footer-part-three">Section headline</h3>
+
+        <ul class="Stack Stack--hasSpacing" style="--stack-spacing: var(--spirit-space-500)">
+          <li>
+            <a href="www.example.com">Link</a>
+          </li>
+          <li>
+            <a href="www.example.com">Link</a>
+          </li>
+          <li>
+            <a href="www.example.com">Link</a>
+          </li>
+          <li>
+            <a href="www.example.com">Link</a>
+          </li>
+          <li>
+            <a href="www.example.com">Link</a>
+          </li>
+          <li>
+            <a href="www.example.com">Link</a>
+          </li>
+        </ul>
+      </nav>
+
+      <nav aria-labelledby="footer-part-four" class="mb-600 mb-tablet-800">
+        <h3 class="typography-body-large-text-bold mb-600" id="footer-part-four">Section headline</h3>
+
+        <ul class="Stack Stack--hasSpacing" style="--stack-spacing: var(--spirit-space-500)">
+          <li>
+            <a href="www.example.com">Link</a>
+          </li>
+          <li>
+            <a href="www.example.com">Link</a>
+          </li>
+          <li>
+            <a href="www.example.com">Link</a>
+          </li>
+        </ul>
+      </nav>
+    </div>
+
+    <hr class="UNSTABLE_Divider mb-700 mb-tablet-400" />
+
+    <ul class="UNSTABLE_Footer__copyright">
+      <li>
+        <a href="www.example.com" class="link-secondary">Legal notice</a>
+      </li>
+      <li>
+        <a href="www.example.com" class="link-secondary">Terms of service</a>
+      </li>
+      <li>
+        <a href="www.example.com" class="link-secondary">Privacy policy</a>
+      </li>
+      <li>
+        <a href="www.example.com" class="link-secondary">Manage cookies</a>
+      </li>
+    </ul>
+  </div>
+</footer>
+```
+
+### Footer Columns Variation
+
+In case you need a specific number of columns for each row in the Footer Columns section with links,
```

Comment:
do we have something called `Footer Columns`?

Reply:
Currently not, but I want to stick with naming above. Maybe it should be `Footer columns`.

Anyway, I would fix it in the #1546.